.gitlab-ci-black-format-code.yml

```diff
@@ -0,0 +1,73 @@
+# .gitlab-ci-black-format-code.yml
```

Comment:
I would explicitly mention pytrhon3 in the filename.

The reason is that we could also have one for the go format (for which we have duplicated code all around the pipelines).